### PR TITLE
WT-2779 Fix large pages getting generated with raw compression

### PR DESCRIPTION
**Pull request by sulabhM**

### CODE REVIEW

File: src/reconcile/rec_write.c [hunk 1]
```diff
@@ -2612,11 +2614,12 @@ __rec_split_raw_worker(WT_SESSION_IMPL *session,
 
 		/*
 		 * Don't create an image so large that any future update will
-		 * cause a split in memory.  Use half of the maximum size so
-		 * we split very compressible pages that have reached the
-		 * maximum size in memory into two equal blocks.
+		 * cause a split in memory. Use the predetermined maximum
+		 * page size for raw compression so we split very compressible
+		 * pages that have reached the maximum size in memory into two
+		 * equal blocks.
```
Comment (michaelcahill) [inline src/reconcile/rec_write.c]: I think the comment "so we split..." belongs with the calculation at line 1986.

**Comment by michaelcahill** [comment]
Note for the record in the commit that this change reverts 8017a9b06cf5fbe941c71c2418401b3dc34f1ab9

**Comment by michaelcahill** [comment]
@sulabhM, I have posted some questions in https://jira.mongodb.org/browse/WT-2779 but this change lgtm and I am happy for it to be merged once the comment I flagged is moved.

**Comment by sulabhM** [comment]
@michaelcahill : I have addressed your comments. Please take a look.

**Comment by michaelcahill** [comment]
Thanks @sulabhM, lgtm.  I'll merge.

**Pull request merged**